### ISSUE TITLE
Implement Auctioneer

### ISSUE THREAD

**Issue by osher** [tikalk-fuse/serverless-rtb#6]
# Implement Auctioneer
Auctioneer is expected to be called with:
```
<host>/auctioneer?au=<adUnitCode>&p=<publisherId>
```
### Assumptions
#7 is done

- well formulated open rtb request:
   - should contain publisher info enriched from data resolved from site-id
   - should contain user info resolved from user-agent, and remote-ip
      (geo-location info from maxmind is not in scope. May be a ***different*** issue).
   - should contain seat info enriched from data resolved from ad-unit-id

## Definition of Done
 - passing test suite emulating all required cases

**Comment by osher** [comment]
depends on #7 kickstart auctioneer
